content/about.ts

```diff
@@ -11,7 +11,7 @@ interface AboutContent {
   body: string[];
   facts: Fact[];
   joinUs: Header;
-  ourPartners: Header;
+  partnersFrame: AirtableEmbed[];
```

Comment:
```suggestion
  partnersFrame: AirtableEmbed;
```
`parternsFrame` indicates that is only one frame, i think remove this `[ ]` is more descriptive.
What do you think?